draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -98,14 +98,14 @@ informative:
 
 --- abstract
 
-Introducing new or modified congestion control algorithms in the global Internet
-has possible ramifications to both the flows using the proposed congestion
-control algorithms and to flows using a standardized congestion control
-algorithm. Therefore, the IETF must proceed with caution when evaluating
-proposals for alternate congestion control. The goal of this document is to
-provide guidance for considering standardization of a proposed congestion
-control algorithm at the IETF. It obsoletes RFC5033 to reflect changes in the
-congestion control landscape.
+This document revises and updates RFC 5033, which discusses the principles and
+guidelines for standardzing new congestion control algorithms. The aim is to
+ensure that proposed congestion control algorithms operate fairly and
```

Comment:
/fairly/ isn't a great choice of words, can we use the formulation for without harm, as we mention later?